### DISCUSSION TITLE
On Debian?

### DISCUSSION THREAD

**Discussion by emresaracoglu** [streamlink/streamlink#4469] [Q&A] [answered]
Hello,

Streamlink works great on windows. So, for the record, can I run it on Debian 10? (not GUI, server version)

```
web2@serve:/var/www/clients/client2$ streamlink --loglevel info https://www.twitch.tv/username best
[cli][info] Found matching plugin twitch for URL https://www.twitch.tv/username 
error: Unable to open URL: https://api.twitch.tv/kraken/users.json (404 Client Error: Not Found for url: https://api.twitch.tv/kraken/users.json?as3=t&login=username)
web2@serve:/var/www/clients/client2$
```
(I actually get the 400 error on the browser, not 404)

Thank you.

**Comment by bastimeyer** [comment, marked as answer]
> Unable to open URL: https://api.twitch.tv/kraken/users.json

You are using a **very much outdated version**. Check the version you're running via [`--loglevel=debug`](https://streamlink.github.io/latest/cli.html#cmdoption-loglevel) or [`--version`](https://streamlink.github.io/latest/cli.html#cmdoption-version).

----

Debian sid - 3.2.0:
https://packages.debian.org/unstable/streamlink

Debian bookworm - 3.2.0:
https://packages.debian.org/bookworm/streamlink

Debian bullseye - 2.0.0:
https://packages.debian.org/bullseye/streamlink

Install docs:
https://streamlink.github.io/latest/install.html#linux-and-bsd

> Please see the [PyPI package and source code](https://streamlink.github.io/latest/install.html#pypi-package-and-source-code) or [AppImages](https://streamlink.github.io/latest/install.html#appimages) sections down below if a package is not available for your distro or platform, or if it's out of date.

**Reply by emresaracoglu** [reply]
**Thank you for your answer,** solved. Have a nice day
